frontend/app/routes/$lang+/_public+/status+/index.tsx

```diff
@@ -178,7 +178,7 @@ export async function action({ context: { session }, params, request }: ActionFu
   const applicationStatusService = getApplicationStatusService();
   const lookupService = getLookupService();
   const { sin, code, firstName, lastName, dateOfBirth } = parsedDataResult.data;
-  const statusId = await applicationStatusService.getStatusId(sin, code, firstName, lastName, dateOfBirth);
+  const statusId = await applicationStatusService.getStatusId({sin: sin, applicationCode: code, firstName: firstName, lastName: lastName, dateOfBirth: dateOfBirth});
```

Comment:
We can use destructuring assignment to avoid repetition of variable names:

```suggestion
  const statusId = await applicationStatusService.getStatusId({sin, applicationCode: code, firstName, lastName, dateOfBirth});
```